app/assets/javascripts/tictactoe.js

```diff
@@ -1 +1,125 @@
-// Code your JavaScript / jQuery solution here
+var board = []
+var turn = 0
+win_combos = [
```

Comment:
Remember to declare your variables -- these will never change, so it's best to do this with `const`